lib/lotus/generators/action.rb

```diff
@@ -0,0 +1,102 @@
+require 'lotus/generators/abstract'
+require 'lotus/utils/string'
+
+module Lotus
+  module Generators
+    class Action < Abstract
+      ACTION_SEPARATOR = /\/|\#/
+      SUFFIX           = '.rb'.freeze
+      TEMPLATE_SUFFIX  = '.html.'.freeze
+      DEFAULT_TEMPLATE = 'erb'.freeze
+
+      def initialize(command)
+        super
+
+        @controller, @action = name.split(ACTION_SEPARATOR)
+        @controller_name     = Utils::String.new(@controller).classify
+        @action_name         = Utils::String.new(@action).classify
+
+        cli.class.source_root(source)
+      end
+
+      def start
+        assert_existing_app!
+
+        opts = {
+          app:           app,
+          controller:    @controller_name,
+          action:        @action_name,
+          template_path: _template_path,
+        }
+
+        templates = {
+          'action.rb.tt' => _action_path,
+          'view.rb.tt'   => _view_path,
+          'template.tt'  => _template_path
+        }
+
+        case options[:test]
+        when 'rspec'
+          templates.merge!({
+            'action_spec.rspec.tt' => _action_spec_path,
+            'view_spec.rspec.tt'   => _view_spec_path,
+          })
+        else
+          templates.merge!({
+            'action_spec.minitest.tt' => _action_spec_path,
+            'view_spec.minitest.tt'   => _view_spec_path,
+          })
+        end
+
+        generate_route
+
+        templates.each do |src, dst|
+          cli.template(source.join(src), target.join(dst), opts)
+        end
+      end
+
+      private
+      def assert_existing_app!
+        unless target.join(app_root).exist?
+          raise Lotus::Commands::Generate::Error.new("Unknown app: `#{ app_name }'")
+        end
+      end
+
+      def generate_route
+        path = target.join(_routes_path)
+        path.dirname.mkpath
+
+        FileUtils.touch(path)
+
+        # Insert at the top of the file
+        cli.insert_into_file _routes_path, before: /\A(.*)/ do
+          "get '/#{ @controller }', to: '#{ name }'\n"
```

Comment:
This is a facility I expect devs to amend this. To generate the correct information is hard. We have two options: 1. CLI arg ( :scream: ) or 2. Interactive CLI.

I think this is a good compromise for now to let us to ship this monday :shipit: